py/desispec/scripts/reformat_proctables.py

```diff
@@ -0,0 +1,177 @@
+"""
+desispec.scripts.updateexptables
```

Comment:
Out of date title (updateexptables vs. reformat_proctables).  In general it is helpful to have the script bin/desi_blah map to the desispec.scripts.blah name, i.e. desi_reformat_processing_tables + desispec.scripts.reformat_processing_tables or desi_reformat_proctables + desispec.scripts.reformat_proctables, but not mix "processing_tables" and "proctables".  I prefer the shorter form for both, but could accept the longer form for both, but let's not mix them.

Reply:
Corrected